### PR TITLE
CSS changes + languages

### PR DESCRIPTION
**Pull request by boombata**
Hi,
Changed a bit the CSS and pages
added language files for french and italian.

Regards,
BM.

### CODE REVIEW

File: advancedopenportal.xml [hunk 1]
```diff
@@ -25,6 +25,8 @@
 	</update>
 	<languages>
                 <language tag="en-GB">language/en-GB/en-GB.com_advancedopenportal.ini</language>
+                <language tag="it-IT">language/it-IT/it-IT.com_advancedopenportal.ini</language>
+
```
Comment (horus68) [inline advancedopenportal.xml]: This is not the proper way to create a language pack on Joomla. 
Each language needs to have its own xml file on a zip folder

**Comment by horus68** [comment]
You can collaborate on translations here:
https://www.transifex.com/rberec/advancedopenportal/

**Comment by horus68** [comment]
@boombata can you clean up your PR to remove the translation files from it?
To install translations you just need to install the zip pack for your language in the Joomla admin. 
Language packs available here https://crowdin.com/project/suitecrmtranslations

**Comment by horus68** [comment]
@boombata also consider to:

- Language values in Language files must be enclosed in quotes, not single quotes.
See: https://docs.joomla.org/Specification_of_language_files
`COM_KEY_CONSTANT="My value"`

- The language file sys.ini should have just the strings for specific admin actions not for administration translation (so usually its a file with fewer keys)

>     a. during the extension's installation, to allow localising the post-installation messages,
>     b. to build the administrator Components menu,
>     c. to localise component parameters and menu parameters
>     d. and in the Extension Manager->Manage.
See: https://docs.joomla.org/Specification_of_language_files

**Comment by boombata** [comment]
Thanks for all your comments, I will clean this PR as soon as possible.

**Comment by horus68** [comment]
@boombata the proper branch for PR to actual version is the Hotfix branch, not the master.
Maybe you should also consider to split the PR in two:
One for the CSS changes and the other only for languages fixes and new strings.
This would be better to avoid collisions with next versions.

I also submitted a PR to Hotfix branch to allow admin translation to the actual version (no collisions with your PR).
After Salesagility merge those PR on languages to the actual version (AOP 2.x) then we should take a look the new 3.x branch

**Pull request closed without merging**